src/controller/mod.rs

```diff
@@ -508,50 +515,6 @@ impl ControllerInner {
         listener
     }
 
-    /// Controls the persistence mode, and parameters related to persistence.
-    ///
-    /// Three modes are available:
-    ///
-    ///  1. `DurabilityMode::Permanent`: all writes to base nodes should be written to disk.
-    ///  2. `DurabilityMode::DeleteOnExit`: all writes are written to disk, but the log is
-    ///     deleted once the `Controller` is dropped. Useful for tests.
-    ///  3. `DurabilityMode::MemoryOnly`: no writes to disk, store all writes in memory.
-    ///     Useful for baseline numbers.
-    ///
-    /// `queue_capacity` indicates the number of packets that should be buffered until
-    /// flushing, and `flush_timeout` indicates the length of time to wait before flushing
-    /// anyway.
-    ///
-    /// Must be called before any domains have been created.
-    pub fn with_persistence_options(&mut self, params: PersistenceParameters) {
```

Comment:
This does not appear to be unused: tests fail to compile with it removed.

Reply:
(I believe it is never used *outside* tests currently, and hence we get a warning when compiling the library.)